Content/Console/build.fsx

```diff
@@ -179,7 +272,7 @@ let dotnetRestore _ =
 let dotnetBuild ctx =
     let args =
         [
-            sprintf "/p:PackageVersion=%s" releaseNotes.NugetVersion
+            sprintf "/p:PackageVersion=%s" latestEntry.NuGetVersion
```

Comment:
[Based on my findings](https://github.com/TheAngryByrd/MiniScaffold/pull/175#issuecomment-579547575), I think we can eliminate this line.

Reply:
Agreed, as per my previous comment.

Reply:
Done in dbc1a55.